airbyte-integrations/bases/base-java/src/test/java/io/airbyte/integrations/base/ssh/SshTunnelTest.java

```diff
@@ -4,18 +4,96 @@
 
 package io.airbyte.integrations.base.ssh;
 
+import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.airbyte.integrations.base.ssh.SshTunnel.TunnelMethod;
 import java.nio.charset.StandardCharsets;
 import java.security.KeyPair;
 import java.security.PrivateKey;
 import java.security.PublicKey;
+import java.util.Arrays;
 import org.apache.sshd.common.util.security.SecurityUtils;
 import org.apache.sshd.common.util.security.eddsa.EdDSASecurityProviderRegistrar;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
 
 class SshTunnelTest {
 
+  private static final String SSH_ED25519_PRIVATE_KEY = "-----BEGIN OPENSSH PRIVATE KEY-----\\n"
```

Comment:
Should this information be placed into a separate file? The gut feeling is that maybe this shouldn't just be openly visible in a test file. There are private keys stored in separate files: https://github.com/airbytehq/airbyte/blob/7aa7a373db6d9072d2642b57dbdb42ef47f92234/tools/integrations-test-ssl/mssql.key

Reply:
These are all random keys created with `ssh-keygen -t rsa` and `ssh-keygen -t ed25519`. 
No actual secrets.

Reply:
Added link do doc